tests/unit/package_managers/yarn/test_resolver.py

```diff
@@ -1,15 +1,28 @@
 import json
-from typing import Any
+import re
+import zipfile
+from pathlib import Path
+from typing import Any, NamedTuple, Optional
 from unittest import mock
 
 import pytest
 
-from cachi2.core.errors import UnsupportedFeature
+from cachi2.core.errors import PackageRejected, UnsupportedFeature
+from cachi2.core.models.sbom import Component
 from cachi2.core.package_managers.yarn.locators import parse_locator
-from cachi2.core.package_managers.yarn.resolver import Package, resolve_packages
+from cachi2.core.package_managers.yarn.project import PackageJson, Project, YarnRc
+from cachi2.core.package_managers.yarn.resolver import Package, create_components, resolve_packages
 from cachi2.core.rooted_path import RootedPath
 
 
+def setup_module() -> None:
```

Comment:
Just for my education: what test is actually disabling the logs?

Reply:
By "bisecting" the test file, I found out that it's something in test_cli.py. But I don't know what specifically and why :disappointed: